packages/cli/src/benchmark.ts

```diff
@@ -0,0 +1,265 @@
+import type {RenderMediaOptions} from '@remotion/renderer';
+import {getCompositions, renderMedia} from '@remotion/renderer';
+import path from 'path';
+import {chalk} from './chalk';
+import {Log} from './log';
+import {makeProgressBar} from './make-progress-bar';
+import {parsedCli, quietFlagProvided} from './parse-command-line';
+import {createOverwriteableCliOutput} from './progress-bar';
+import {bundleOnCli} from './setup-cache';
+
+const DEFUALT_RUNS = 3;
+const DEFAULT_COMP_ID = 'Main';
+const DEFAULT_FILE_PATH = './src/index';
```

Comment:
Let's make the user explicitly pass the path: `npx remotion benchmark src/index.tsx ...` for now.
We will tackle this in another issue where this can be omitted for all commands.

Reply:
So we don't need to download (or clone) benchmark repo?